editor.html

```diff
@@ -297,6 +309,12 @@ <h2 id="modalTitle">Export</h2>
             </label>
           </div>
 
+          <div>
+            <a href="#" class="dialog-action-link" id="showPreviewArrayIcons">Show JSON</a>
```

Comment:
I'd like to use the `<details>`/`<summary>` elements instead to do the open/close without Javascript.